src/api/RequestApi.ts

```diff
@@ -64,7 +78,7 @@ const transformInRequestToSP = (request: IInRequest): IRequestItem => {
     CACExpiration: request.CACExpiration
       ? request.CACExpiration.toISOString()
       : "",
-    supGovLeadId: request.supGovLead.SPUserId as number, // TODO: forcing as number becasue type says optional but we are requiring
+    supGovLeadId: request.supGovLead.SPUserId as number, // forcing as number becasue type says optional but we are requiring
```

Comment:
Should have made my TODO note here a little better.  By saying "as number" I am telling the Typescript compiler to ignore the fact that it detected that SPUserId may be undefined and not a number and supGoveLeadId must be a number.  Ideally we fix up the types to handle it properly as to when this Id is required and when it is not.
I added it as a note to Issue #12 for when we standardize/cleanup the different "people" types.